float32 latitude 
float32 longitude
float32 altitude
float32 v_ns
float32 v_ew

